### PR TITLE
Mobile first

### PR DESCRIPTION
**Pull request by JansueT**
For this project, I have implicated the below step;

- For this project, I have used the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
- The 2 sections I builded for this project are:
     - The toolbar (or header)
     - The headline section (right after the header)
- In order to lay out the elements on the page you should use Flexbox in all 2 sections.
- I don't need to implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.
- I stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates 3 in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  </head>
+</html>
+<body>
+  <header class="header">
+    <nav class="header_nav">
+      <div class="header_logo black"><a class="black" href="#">Welcome</a></div>
+      <div class="header_menu">
+        <img class="menu_icon" src="images/Union.png" alt="menu" />
+      </div>
+    </nav>
```
Comment (iambenkis) [inline index.html:30]: - [ ] Good job @JansueT :+1: , however the link to the hamburger image is broken. Kindly provide a proper url :pray: 

Your design
 
Screenshot from 2022-12-02 19-31-28

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        </li>
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/medium.png" alt="M" />
+          </a>
+        </li>
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/angellist.png" alt="Hand" />
+          </a>
```
Comment (iambenkis) [inline index.html:74]: - [ ] Good job @JansueT :+1: , however the link to the hamburger image is broken. Kindly provide a proper url :pray: 

Your design

Screenshot from 2022-12-02 19-31-46 (1)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <br>Cansu</span>
+      <span class="sub-headline_title">I'm a software 
+        <br>developer</span>
+    </h1>
+    <p class="headline_content">
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
+      need coded, don’t hestiate to contact me. </p>
+    <div class="backimg"></div>
+    <div class="addline_social">
```
Comment (iambenkis) [inline index.html:44]: - [ ] I appreciate the effort you put in implementing your design, but your design should stick as much as possible to the Figma design :100:. That means you should make sure to check if the attribute such as ```font-family```, ```color```, ```margin```, ```font-weight``` etc are the same in your design as in the Figma design. :smile: 
- [ ] Kindly make sure the ```font-family``` look the same in your design as given in the Figma design :health_worker: 

Your design:

Screenshot from 2022-12-02 19-46-32

The Figma design

Screenshot from 2022-12-02 19-47-13

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @JansueT 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️

- [ ] You should stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Debas-31 — APPROVED** [review]
> ## Hi @JansueT  :wave:, 

# Status: Approved :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Figma template usage :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — COMMENTED** [review]
# Approved :green_circle: 
Hi @JansueT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] No linter error :white_check_mark: 
- [ ] GitHub flow is used correctly :white_check_mark: 
- [ ] Descriptive pull request summary :white_check_mark: 

## Optional suggestions
- [ Optional ] Please consider adding a `License` file in your project directory and also add a link to the License file on the `readme` file. It's considered good practice to always have a License in your project to inform people who would like to use your code how it is Licensed. Once you've added the link to the License, you can remove the `NOTE` section under the License link.

Screenshot from 2023-02-18 13-51-26

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — COMMENTED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @JansueT 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- [ ] [ Optional ] Please consider adding a License file in your project directory and also add a link to the License file on the readme file. It's considered good practice to always have a License in your project to inform people who would like to use your code how it is Licensed. Once you've added the link to the License, you can remove the NOTE section under the License link.
- _Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**